string world_builder_file
string gazebo_model_path
string noise_file
---